### PR TITLE
WOE 22 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/faerie_slumber_party.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Faerie Slumber Party
+ManaCost:4 U U
+Types:Sorcery
+A:SP$ ChangeZoneAll | ChangeType$ Creature | Origin$ Battlefield | Destination$ Hand | RememberChanged$ True | SubAbility$ DBToken | SpellDescription$ Return all creatures to their owners' hands.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/faerie_slumber_party.txt:4]: RememberLKI ⚠️

File: forge-gui/res/cardsfolder/upcoming/faerie_slumber_party.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+SVar:DBToken:DB$ Token | TokenAmount$ Y | TokenScript$ u_1_1_faerie_flying_cantblock | TokenOwner$ You | SubAbility$ DBCleanup | SpellDescription$ For each opponent who controlled a creature returned this way, you create two 1/1 blue Faerie creature tokens with flying and "This creature can block only creatures with flying."
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:Y:SVar$X/Times.2
+SVar:X:PlayerCountOpponents$ConditionGE1 Creature.IsRemembered
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/faerie_slumber_party.txt:8]: Try `PlayerCountRememberedController$HasPropertyOpponent`

File: forge-gui/res/cardsfolder/upcoming/ego_drain.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Ego Drain
+ManaCost:B
+Types:Sorcery
+A:SP$ Discard | ValidTgts$ Player | NumCards$ 1 | DiscardValid$ Card.nonLand | Mode$ RevealYouChoose | SubAbility$ DBExileHand | SpellDescription$ Target opponent reveals their hand. You choose a nonland card from it. That player discards that card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ego_drain.txt:4]: wrong Tgts

File: forge-gui/res/cardsfolder/upcoming/extraordinary_journey.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Name:Extraordinary Journey
+ManaCost:X X U U
+Types:Enchantment
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Destination$ Battlefield | Execute$ TrigExile | TriggerDescription$ When CARDNAME enters the battlefield, exile up to X target creatures. For each of those cards, its owner may play it for as long as it remains exiled.
+SVar:TrigExile:DB$ ChangeZone | Origin$ Battlefield | Destination$ Exile | TgtPrompt$ Select up to X target creatures | ValidTgts$ Creature | TargetMin$ 0 | TargetMax$ X | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | Duration$ Permanent | StaticAbilities$ MayPlay | RememberObjects$ Targeted | ForgetOnMoved$ Exile | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/extraordinary_journey.txt:6]: cleanup for what?

File: forge-gui/res/tokenscripts/u_1_1_faerie_flying_cantblock.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Name:Faerie Token
+ManaCost:no cost
+Types:Creature Faerie
+Colors:blue
+PT:1/1
+K:Flying
+S:Mode$ CantBlockBy | ValidAttacker$ Creature.withoutFlying | ValidBlocker$ Creature.Self | Description$ CARDNAME can block only creatures with flying.
+Oracle:Flying/nThis creature can block only creatures with flying.
```
Comment (tool4ever) [inline forge-gui/res/tokenscripts/u_1_1_faerie_flying_cantblock.txt:8]: \n

**Comment by Simisays** [comment]
after some testing it seems like Faerie Fencing isn't working

File: forge-gui/res/cardsfolder/upcoming/feral_encounter.txt [hunk 6]
```diff
@@ -0,0 +1,13 @@
+Name:Feral Encounter
+ManaCost:G G
+Types:Sorcery
+A:SP$ Dig | Defined$ You | DigNum$ 5 | ChangeValid$ Creature | DestinationZone$ Exile | RememberChanged$ True | RestRandomOrder$ True | SubAbility$ DBMayPlay | SpellDescription$ Look at the top five cards of your library. You may exile a creature card from among them. Put the rest on the bottom of your library in a random order. You may cast the exiled card this turn. At the beginning of the next combat phase this turn, target creature you control deals damage equal to its power to up to one target creature you don’t control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/feral_encounter.txt:4]: may not handled

File: forge-gui/res/cardsfolder/upcoming/feral_encounter.txt [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+SVar:DBMayPlay:DB$ Effect | StaticAbilities$ STPlay | RememberObjects$ Remembered | ExileOnMoved$ Exile | SubAbility$ DBDelayTrig 
+SVar:STPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered+nonLand | AffectedZone$ Exile | Description$ You may cast the exiled card this turn. 
+SVar:DBDelayTrig:DB$ DelayedTrigger | ThisTurn$ True | Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | Execute$ TrigFight | SubAbility$ DBCleanup | TriggerDescription$ At the beginning of the next combat phase this turn, target creature you control deals damage equal to its power to up to one target creature you don’t control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/feral_encounter.txt:7]: why ValidPlayer?

File: forge-gui/res/cardsfolder/upcoming/food_coma.txt [hunk 8]
```diff
@@ -0,0 +1,9 @@
+Name:Food Coma
+ManaCost:3 W
+Types:Enchantment
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Destination$ Battlefield | Execute$ TrigExile | TriggerDescription$ When CARDNAME enters the battlefield, exile target creature an opponent controls until Food Coma leaves the battlefield. Create a Food token. (It's an artifact with "{2}, {T}, Sacrifice this artifact: You gain 3 life.")
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/food_coma.txt:4]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/food_coma.txt [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+SVar:TrigExile:DB$ ChangeZone | ValidTgts$ Permanent.nonLand+OppCtrl | TgtPrompt$ Select target nonland permanent an opponent controls | SubAbility$ DBFood | Origin$ Battlefield | Destination$ Exile | Duration$ UntilHostLeavesPlay
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/food_coma.txt]: Wrong Tgt

File: forge-gui/res/cardsfolder/upcoming/food_coma.txt [hunk 10]
```diff
@@ -0,0 +1,9 @@
...
+SVar:DBFood:DB$ Token | TokenAmount$ 2 | TokenScript$ c_a_food_sac | TokenOwner$ You
```
[the remark this answers is not in the document]
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/food_coma.txt:6]: hmm I fixed this in a follow up PR. Seems like Github is acting weird ? https://github.com/Card-Forge/forge/pull/3711/commits/08f43cce8a153dd1f288fd9a2aacc881ac900c15

File: forge-gui/res/cardsfolder/upcoming/faerie_fencing.txt [hunk 11]
```diff
@@ -0,0 +1,10 @@
+Name:Faerie Fencing
+ManaCost:X B
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ -X | NumDef$ -X | IsCurse$ True | SpellDescription$ Target creature gets -X/-X until end of turn. That creature gets an additional -3/-3 until end of turn if you controlled a Faerie as you cast this spell.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/faerie_fencing.txt:4]: maybe it works if you use the right SVar

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/faerie_fencing.txt:4]: oops 🤦

File: forge-gui/res/cardsfolder/upcoming/food_fight.txt [hunk 12]
```diff
@@ -0,0 +1,9 @@
+Name:Food Fight
+ManaCost:1 R
+Types:Enchantment
+S:Mode$ Continuous | Affected$ Artifact.YouCtrl | AddAbility$ SacMe | Description$ Artifacts you control have "{2}, Sacrifice this artifact: It deals damage to any target equal to 1 plus the number of permanents named Food Fight you control."
+SVar:SacMe:AB$ DealDamage | Cost$ 2 Sac<1/CARDNAME> | ValidTgts$ Any | NumDmg$ X | SpellDescription$ It deals damage to any target equal to 1 plus the number of permanents named Food Fight you control.
+DeckNeeds:Type$Artifact & Name$Food Fight
+DeckHas:Ability$Sacrifice
+SVar:X:Count$Valid Card.namedFood Fight/Plus.1
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/food_fight.txt]: YouCtrl

File: forge-gui/res/cardsfolder/upcoming/frantic_firebolt.txt [hunk 13]
```diff
@@ -0,0 +1,7 @@
+Name:Frantic Firebolt
+ManaCost:2 R
+Types:Instant
+A:SP$ DealDamage | ValidTgts$ Any | NumDmg$ X | SpellDescription$ CARDNAME deals X damage to target creature, where X is 2 plus the number of cards in your graveyard that are instant cards, sorcery cards, and/or have an Adventure.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/frantic_firebolt.txt:4]: Wrong Tgt

**Review by tool4ever — APPROVED** [review]

**Pull request merged**